app/templates/frameworks/contract_variation.html

```diff
@@ -28,16 +28,16 @@
     <div class="grid-row">
       <div class="column-one-whole">
         {% if errors %}
-         {% with 
+         {% with
             message = "You must <a href=\"#{}\">accept the changes</a> to continue.".format(errors.accept_changes.input_name)|safe, type = "destructive" %}
```

Comment:
Double quotes inside double quotes here (my editor's gone red!)

Reply:
Yikes! I think the backslash should handle this? I don't think I actually touched that line...